### PR TITLE
[add-notes] update local.tf file

### PR DESCRIPTION
**Pull request by aliartiza75**
# PR o'clock

## Description

I have updated `local.tf` file based on the discussion of this [issue](https://github.com/terraform-aws-modules/terraform-aws-eks/issues/510)

### Checklist

- [x] `terraform fmt` and `terraform validate` both work from the root and `examples/*` directories
- [x] CI tests are passing
- [ ] I've added my change to CHANGELOG.md and highlighted any breaking changes
- [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by aliartiza75** [comment]
@max-rocket-internet kindly review this PR and let me know if you want me to modify anything.

File: local.tf [hunk 1]
```diff
@@ -23,9 +23,9 @@ locals {
...
+    asg_desired_capacity          = "1"                         # Desired worker capacity in the autoscaling group and changing its value will not affect the autoscaling group's desired capacity because the cluster-autoscaler manages up and down scaling of the nodes. Cluster-autoscaler add nodes when pods are in pending state and remove the nodes when they are not required by modifying the desirec_capacity of the autoscaling group. Although an issue exists in which if the value of the asg_min_size is changed it modifies the value of asg_desired_capacity.
     asg_max_size                  = "3"                         # Maximum worker capacity in the autoscaling group.
-    asg_min_size                  = "1"                         # Minimum worker capacity in the autoscaling group.
+    asg_min_size                  = "1"                         # Minimum worker capacity in the autoscaling group. NOTE: Change in this paramater will affect the asg_desirec_capacity, like changing its value to 2 will change asg_desired_capacity value to 2 but bringing back it to 1 will not affect the asg_desired_capacity.
```
Comment (barryib) [inline local.tf]: typo : `asg_desirec_capacity ` => `asg_desired_capacity`

Comment (aliartiza75) [reply local.tf]: @barryib typo has been fixed

**Comment by barryib** [comment]
@aliartiza75 From https://github.com/terraform-aws-modules/terraform-aws-eks/issues/510#issuecomment-531712944

> Add a table of variables in the main README.md because in main README.md no documentation exists. is it fine from you side?

Is ok now. 

> I will also update the auto_scalaing doc as you suggested

It sounds like you plan to update the [auto_scalaing doc](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/autoscaling.md) also. Can you do this in this PR to close https://github.com/terraform-aws-modules/terraform-aws-eks/issues/510

**Comment by aliartiza75** [comment]
@barryib I have updated the documents kindly re-review it thank you.

**Pull request merged**